contracts/BugBounty.sol

```diff
@@ -30,4 +27,9 @@ contract BugBounty {
   function resolveClaim(bytes32 _claimHash, uint payout) onlyOwner {
 
   }
+
+  function getOwner() returns (address) {
+    return owner;
+  }
+
```

Comment:
@cwhinfrey I removed this and the `owner` property. it's provided by `Ownable` and make public there